### ISSUE TITLE
regression (test an image : wrong prediction score)- ground truth 0.66 but testing time return 11.153

### ISSUE THREAD

**Issue by ranju2015** [NVIDIA/DIGITS#1948]
Hi,

I am doing some experiment to solve a regression problem.

My train and validation data looks like the following ( image name and one score value)
61086.jpg 0.42
10168.jpg 0.43
745781.jpg 0.39
76352.jpg 0.39
803290.jpg 0.66 

So, I prepared custom lmdb dataset for train image, train label, validation image and validation score. 

My training prototxt file contains layer Eucleadian loss layer as suggested by digits regression example 

layer {
  name: "loss/classifier"
  type: "InnerProduct"
  bottom: "FC_concat"
  top: "loss/classifier"
  param {
    lr_mult: 1.0
    decay_mult: 1.0
  }
  param {
    lr_mult: 2.0
    decay_mult: 0.0
  }
  inner_product_param {
    num_output: 1
    weight_filler {
      type: "xavier"
    }
    bias_filler {
      type: "constant"
      value: 0.0
    }
  }
}
layer {
  name: "loss"
  type: "EuclideanLoss"
  bottom: "loss/classifier"
  bottom: "label"
  top: "loss"
  exclude {
    stage: "deploy"
  }
}

My training train curve is smooth and very low error rate.
![image](https://user-images.githubusercontent.com/14028800/35180590-67510c36-fdff-11e7-86d6-764e41b045e1.png)

Please suggest me a solution. So that I get the score between 0-1 range ( as my training score).

**Comment by gheinrich** [comment]
Hello, did you verify that you applied the same pre-processing to your test images as you did to your training/validation images? You might find it useful to enable "visualizations" at test time. Check the distribution of your input data and make sure this matches that of the training data.
